### PR TITLE
qemu-img supports coroutines complete

### PR DESCRIPTION
**Pull request by hereischen**
Signed-off-by: Haotong Chen <hachen@redhat.com>

id: 1695412

### CODE REVIEW

File: qemu/tests/qemu_img_supports_coroutines_complete.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        """Run qemu-io cmd to a given img."""
+        logging.info("Run qemu-io %s" % img.image_filename)
+        q = QemuIOSystem(test, params, img.image_filename)
+        q.cmd_output(cmd, 120)
+
+    def _create_error_cfg():
+        logging.info("Create error cfg /tmp/blkdebug.cfg.")
+        error_cfg = ('[inject-error]\nevent = "write_aio"\n'
+                     'sector = "819200"\nonce = "on"')
+        with open("/tmp/blkdebug.cfg", "w") as cfg:
```
Comment (lolyu) [inline qemu/tests/qemu_img_supports_coroutines_complete.py]: better to delete `blkdebug.cfg` after case-finish.

File: qemu/tests/cfg/qemu_img_supports_coroutines_complete.cfg [hunk 2]
```diff
@@ -0,0 +1,14 @@
+- qemu_img_supports_coroutines_complete:
```
Comment (hellohellenmao) [inline qemu/tests/cfg/qemu_img_supports_coroutines_complete.cfg]: Should add "convert" in the name, thanks.

File: qemu/tests/qemu_img_supports_coroutines_complete.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+from virttest.qemu_io import QemuIOSystem
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    qemu-img supports coroutines complete.
+
+    1. create source and target images, write 0 into
+       the entire source image through qemu-io
```
Comment (hellohellenmao) [inline qemu/tests/qemu_img_supports_coroutines_complete.py]: entire? From the log file, I noticed the virtual size of the tested image is 20G. We just wrote 1G data to it.

File: qemu/tests/qemu_img_supports_coroutines_complete.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            cfg.write(error_cfg)
+
+    def _inject_error_and_verify(file):
+        try:
+            pkg = utils_package.LocalPackageMgr("valgrind")
+            pkg.install()
+            _create_error_cfg(file)
+            cmd = ("valgrind --soname-synonyms=somalloc=libtcmalloc.so"
+                   "qemu-img convert -npWO qcow2 source.qcow2"
+                   "blkdebug:%s:target.qcow2" % file)
```
Comment (hellohellenmao) [inline qemu/tests/qemu_img_supports_coroutines_complete.py]: There should be a " " at the beginning, thanks.

**Pull request merged**